site/book/05-developing-functions/04-developing-in-Starlark.md

```diff
@@ -0,0 +1,56 @@
+You can write your functions in Starlark script.
+
+?> Starlark SDK is in *experimental* stage.
+
+[Starlark] is a python-like language designed for use in configuration files
+that has several desirable properties: deterministic evaluation, hermetic
+execution, and simplicity. The `starlark` function contains the interpreter for
+the language, and accepts a `functionConfig` of kind `StarlarkRun` where you
+provide your business logic.
+
+Current Starlark is driven by Go function [starlark](https://catalog.kpt.dev/starlark/v0.4).
```

Comment:
> Go function

No need to mention what language it used.

Reply:
I'll list the full image then. (that's also the point where myself got confused get the first sight